src/index.js

```diff
@@ -274,6 +274,34 @@ class PublicationClient extends EventEmitter {
     return subscription;
   }
 
+  /**
+   * Subscribe to the publication with the given name. Any other parameters
+   * are passed as arguments to the publication.
+   *
+   * @param {String} name The publication to subscribe to.
+   * @param {Object} options containing optional parameters such as bootstrap and/or
+   * bootstrapAsync.
+   * @param {*[]} params (optional) Params to pass to the publication.
+   * @returns {Subscription} The subscription to the desired publication.
+   */
+  subscribeWithOptions(name, options, ...params) {
+    const id = this._nextSubscriptionId++;
+    const payload = [...params, options];
```

Comment:
It seems like an alternative would be to pass options as the last argument to the `subscribe` call? How do you think about adding this new method vs. just calling `subscribe` with a different last argument?

Reply:
the only reason is readability - I felt that it would be much cleaner to leave the old method intact and to introduce new one. As not all of the clients will move to the new version, it seemed cleaner. No other reason - and if this is opposite to the standard practices - I'm more then happy to merge those methods.
Of course, I will create follow up ticket to clear the old code and to migrate functionality once everything is tested.